packages/charting/src/components/StackedBarChart/StackedBarChart.styles.ts

```diff
@@ -60,6 +60,20 @@ export const getStyles = (props: IStackedBarChartStyleProps): IStackedBarChartSt
       fontSize: FontSizes.small,
       color: theme.palette.black,
       opacity: '0.6'
+    },
+    benchmarkContainer: {
+      position: 'relative',
+      height: '11.8px'
+    },
+    triangle: {
+      position: 'absolute',
+      width: '0',
+      height: '0',
+      borderLeft: '4.5px solid transparent',
+      borderRight: '4.5px solid transparent',
+      borderTop: '7.8px solid',
+      borderTopColor: theme.palette.neutralTertiary,
```

Comment:
@mltejera should we find a semantic slot here that makes sense?

Reply:
sure - @luckyrass - do you mind sharing a screenshot calling out this element in particular with some context around it? That helps us figure out the right color.

Reply:
@mltejera Below are the screenshots you need.

- benchmark triangle refer to the Industry benchmark score.

![image](https://user-images.githubusercontent.com/12230173/65207673-f461b500-dac4-11e9-8fc3-edf48383f7b6.png)

- target triangle refer to the target score of this tenant.

![image](https://user-images.githubusercontent.com/12230173/65207588-9f25a380-dac4-11e9-800f-1c9b3de4ed98.png)

Reply:
Ah - I see now. I saw 'border' and know that we have half a dozen different border semantic colors. This is more of a graphical decoration, so a theme palette color is the way to go on this one.

Apologies for raising the alarm.